filesystem/profile

```diff
@@ -24,41 +24,42 @@
 MSYS2_PATH="/usr/local/bin:/usr/bin:/bin"
 MANPATH="/usr/local/man:/usr/share/man:/usr/man:/share/man:${MANPATH}"
 INFOPATH="/usr/local/info:/usr/share/info:/usr/info:/share/info:${INFOPATH}"
-if [ -n "$SET_FULL_PATH" ]
-then
-  WIN_PATH="${PATH}"
-else
-  WIN_ROOT="$(PATH=${MSYS2_PATH} cygpath -Wu)"
-  WIN_PATH="${WIN_ROOT}/System32:${WIN_ROOT}:${WIN_ROOT}/System32/Wbem:${WIN_ROOT}/System32/WindowsPowerShell/v1.0/"
-fi
+
+case "${MSYS}" in
+  *winpath:none*) unset WIN_PATH ;;
```

Comment:
Really don't do it. Empty windows path will lead to not working msys2 as you will not have system libraries in the PATH